Emails/kyc-rejected-email/content-part/en.html

```diff
@@ -0,0 +1,7 @@
+<p style="margin: 0; padding: 0;">Dear ${AdminUserName},</p>
+<br>
+<p style="margin: 0; padding: 0;">We regret to inform you that your MAVN verification for ${BusinessName} has been rejected with the following note as to why the application was rejected.</p>
+<br>
+<p style="margin: 0; padding: 0;">${RejectionComment}</p>
+<br>
+<p style="margin: 0; padding: 0;">Please contact our support team at support@mavn.ch for further queries on your application.</p>
```

Comment:
support email should be
`<a href="mailto:${CommonInfo::SupportEmailAddress}">${CommonInfo::SupportEmailAddress}</a>`